packages/fast-usdc-contract/test/exos/settler.test.ts

```diff
@@ -807,6 +825,10 @@ test('slow path, and forward fails (terminal state)', async t => {
     ...defaultSettlerParams,
   });
 
+  // FIXME this rejection isn't being caught within the flow
+  // The console shows SES_UNHANDLED_REJECTION for this error
+  // and the await within the flow succeeds when it should throw.
```

Comment:
FIXME when?
I got the impression that you found a solution to this.

Reply:
oops, leftover from debugging. will remove